src/components/course/data/tests/hooks.test.jsx

```diff
@@ -1116,33 +904,55 @@ describe('useUserSubsidyApplicableToCourse', () => {
 
   beforeEach(() => {
     jest.clearAllMocks();
+    useParams.mockReturnValue({ courseKey: 'edX+DemoX' });
+    useEnterpriseCustomer.mockReturnValue({ data: mockEnterpriseCustomer });
+    useCourseMetadata.mockReturnValue({});
```

Comment:
[clarification] Is `useCourseMetadata` used within `useUserSubsidyApplicableToCourse`?

Reply:
It is used through `useCourseListPrice`. Attempting just to mock `useCourseListPrice` still results in the following error 
![Screenshot 2024-04-11 at 9 53 30 AM](https://github.com/openedx/frontend-app-learner-portal-enterprise/assets/82611798/bf4f23f9-bc4e-4609-bb18-bb7e49c33f75)

Since the return from `useCourseMetadata` is equivalent to the return of `useCourseListPrice`, mocking `useCourseMetadata` resolves the error.

Reply:
Ah, thanks for clarifying. I just tried mocking `useCourseListPrice` and am running into the same issue as your screenshot above. Also tried using `jest.spyOn`. Let's keep as is.